manifest.json

```diff
@@ -21,6 +21,14 @@
         "48": "icons/icon-48.png"
       },
       "default_popup": "popup/popup.html"
+    },
+    "commands": {
+      "_execute_browser_action": {
+        "suggested_key": {
+          "default": "Ctrl+Shift+Y"
```

Comment:
I tried using Ctrl+Shift+J but I guess you can't use combinations that are already reserved for Firefox, which is an awful lot.